HowToUseOCR.md

```diff
@@ -3,9 +3,11 @@
 This document provides instructions on how to add additional language packs for the OCR tab in Stirling-PDF, both inside and outside of Docker.
 
 ## My OCR used to work and now doesn't!
-The paths have changed for the tessadata locations on new docker images, please use ``/usr/share/tessdata`` (Others should still work for backwards compatability but might not)
+
+The paths have changed for the tessadata locations on new docker images, please use ``/languages`` (Others should still work for backwards compatability but might not)
```

Comment:
This seems like a breaking change and doesn't seem backwards compatible.

Reply:
Ignore this comment if it's not.

Reply:
> breaking change

Yes, it is. From my perspective it's the only way to separate the files provided by the user from the system files.

> backwards compatible

What kind of compatibility are you thinking of here? Or maybe I'm confused about a possible backwards direction.

Reply:
Then we should remove the part in the parentheses to reflect that.

Reply:
Oh, I think I misunderstood something fundamental. I thought `/usr/share/tessdata` is used by the system packages. But you have introduced this already as an artificial target folder for the languages. I'll rework my PR to reflect that properly.

Reply:
This is getting tricky. The packaging seems to be different between Alpine and Debian. On Debian, `/usr/share/tessdata` isn't being used: https://packages.debian.org/bookworm/all/tesseract-ocr-aze/filelist. But on Alpine this is the target for the language files. That's why I'd still suggest separating this folder used by the package manager by the folder with the user provided languages `/languages`. This would mean a break for the Docker users. But the advantage is that no files need to be preserved or copied, and thus the user isn't able to break the whole feature.

Reply:
> Then we should remove the part in the parentheses to reflect that.

Just did that.